### PR TITLE
Adding a prefix when date is uncertain

### PR DESCRIPTION
**Pull request by cgalarza**
For uncertain dates, we'd like to support displaying something like, "possibly 2002". Approximate dates have a prefix and suffix, so I followed that example. I am not sure what the best way to test this is, let me know if you have any advice on that front. Thanks!

### CODE REVIEW

**Comment by corylown** [comment]
@cgalarza thank you for this proposed contribution. I'm happy to include this change and will release a new version once it's merged. Before I merge it, could you please add translation entries for `uncertain_date_prefix` for the other languages as well? This will help anyone wanting to use these languages to understand the options that are available to them. I'm not sure of a good way to test this since the default translation is empty.

**Comment by cgalarza** [comment]
@corylown Whoops! I should have remembered to do that. I added in those translations. Just a heads up that I have another change/request in the works, in case you want to hold off on doing a release until after. Thanks for your help!

**Review by corylown — APPROVED** [review]

**Pull request merged**
